config/config-mysql_apache_jvm_statsd.yaml

```diff
@@ -214,16 +243,26 @@ processors:
       
       - metric_name: apache_sent_kilobytes_total
         action: update
-        new_name: agent.googleapis.com/apache/traffic
+        new_name: apache/traffic
         operations:
           - action: toggle_scalar_data_type
 
+  filter/host:
+    metrics:
+      exclude:
+        match_type: strict
+        metric_names:
+          - system.network.dropped_packets
+          - system.swap.usage
+          - system.filesystem.inodes.usage
+          - system.swap.page_faults
+
   metricstransform/host:
     transforms:
```

Comment:
Is there a reason we only have a few metrics here and not the whole list? (can copy from the config-linux file)

Reply:
oh yeah, must not have taken the latest version by mistake, fixed!